docs/fastdds/xml_configuration/example.rst

```diff
@@ -0,0 +1,14 @@
+.. _examplexml:
+
+Example
+-------
+
+In this section, there is a full XML example with all possible configuration.
+It can be used as a quick reference, but it may not be valid due to incompatibility or exclusive properties.
+Do not take it as a working example.
+
+.. literalinclude:: /../code/XMLTester.xml
+    :language: xml
+    :start-after: <!-->XML-EXAMPLE<-->
+    :end-before: <!--><-->
+    :lines: 2-3,5-
```

Comment:
Double check that all options are present. The XML is not well formed, it should end in `</profiles>`